installation/resources/installer-cr.yaml.tpl

```diff
@@ -26,8 +26,8 @@ spec:
       namespace: "knative-serving"
     - name: "knative-eventing"
       namespace: "knative-eventing"
-    # - name: "prometheus-operator"
-    # namespace: "kyma-system"
+    - name: "prometheus-operator"
```

Comment:
you are changing the default installation behaviour of kyma here, is that intended?

Reply:
Here is the dependency chain looks like: `kiali` -> `monitoring` -> `prometheus-operator`